orangegrove.rb

```diff
@@ -0,0 +1,85 @@
+# ORANGE TREE ------------------------------------------------------------------------------->
+class OrangeTree
+
+	attr_reader :height, :age, :orange_count, :max_age, :dead
+
+	def initialize
+		@height = 0
+		@age = 0
+		@orange_count = 0
+		@max_age = 100
+		@dead = false
+	end
+
+	# Ages tree one year, grows tree 0.75ft/year, increases oranges/tree after age 3
+	# Determines if it's the tree's time to shuffle off the mortal coil
+	def one_year_passes
+		@orange_count = 0
```

Comment:
Since you're setting `@orange_count` on lines 21 and 26, you can skip setting it here. Or, you could set it here and then only change it if your tree is older than 3 years.